examples/all-clusters-app/linux/fuzzing-main.cpp

```diff
@@ -73,7 +73,11 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t * aData, size_t aSize)
     // For now, just dump the data as a UDP payload into the session manager.
     // But maybe we should try to separately extract a PeerAddress and data from
     // the incoming data?
-    Transport::PeerAddress peerAddr;
+
+    // dumping payload with random transport types
+    Transport::Type fuzzedTransportType = static_cast<Transport::Type>(*aData % 5);
```

Comment:
This will always have the transport type match the first byte of the data.  Shouldn't we be using only the parts of the data starting with index 1 for the actual data, to decouple those?

Reply:
Done